app/uk/gov/hmrc/cipphonenumberstubs/dao/OtpCacheRepository.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cipphonenumberstubs.dao
+
+import play.api.Configuration
+import uk.gov.hmrc.mongo.{MongoComponent, TimestampSupport}
+import uk.gov.hmrc.mongo.cache.{CacheIdType, MongoCacheRepository}
+
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.FiniteDuration
+
+class OtpCacheRepository @Inject()(mongoComponent: MongoComponent,
+                                   configuration: Configuration,
+                                   timestampSupport: TimestampSupport)(implicit ec: ExecutionContext)
+  extends MongoCacheRepository(
```

Comment:
We should be using the **[DefaultPlayMongoRepositorySupport](https://github.com/hmrc/hmrc-mongo/tree/main/hmrc-mongo-test-play-28)** as this allows us to setup and tear down after tests. That way we don't have a Mongocache that stays on even when tests are not running

Reply:
ok i will make another ticket

Reply:
please see https://jira.tools.tax.service.gov.uk/browse/CAV-379
for the tech debt ticket to find a better solution.